pythnet/pythnet_sdk/src/messages.rs

```diff
@@ -50,20 +51,23 @@ use {
 pub enum Message {
     PriceFeedMessage(PriceFeedMessage),
     TwapMessage(TwapMessage),
+    PublisherStakeCapsMessage(PublisherStakeCapsMessage),
 }
 
 impl Message {
     pub fn publish_time(&self) -> i64 {
         match self {
             Self::PriceFeedMessage(msg) => msg.publish_time,
             Self::TwapMessage(msg) => msg.publish_time,
+            Self::PublisherStakeCapsMessage(msg) => msg.publish_time,
         }
     }
 
     pub fn feed_id(&self) -> FeedId {
         match self {
             Self::PriceFeedMessage(msg) => msg.feed_id,
             Self::TwapMessage(msg) => msg.feed_id,
+            Self::PublisherStakeCapsMessage(_) => [0u8; 32],
```

Comment:
Maybe we should change the function to return `Option<FeedId>` since there is no feed id for this kind of message.

Reply:
addressed

Reply:
I think the api won't be clean in this way and at some point we should fix it. It's fine now as we are not using this on the validator. Let's also add a TODO to mention that we should fix it. i also recommend using something other than [0; 32] for it to distinguish it from empty/default.

Reply:
[0;32] is good because it's guaranteed to not conflict with any price feed ids